stored_requests/events/http/http.go

```diff
@@ -40,6 +40,10 @@ import (
 //   "imps": {
 //     "imp1": { ... stored data for imp1 ... },
 //     "imp2": { ... stored data for imp2 ... },
+//   },
+//   "responses": {
+//     "resp1": { ... stored data for resp1 ... },
+//     "resp2": { ... stored data for resp2 ... },
```

Comment:
Is this commented out code still needed?

Reply:
Yes, this is a valid comment.